pkg/crc/systemd/actions/action.go

```diff
@@ -23,6 +24,7 @@ func (action Action) String() string {
 		"disable",
 		"status",
 		"daemon-reload",
+		"enable --now",
```

Comment:
Not a big fan of pretending `enable --now` is an 'action'. This is leaking at the wrong level that we use actions as command line arguments. 
In theory, `stdOut, stdErr, err = c.commandRunner.Run("systemctl", "enable --now", name)` should be `stdOut, stdErr, err = c.commandRunner.Run("systemctl", "enable", "--now", name)`